### ISSUE TITLE
[FEATURE REQUEST] Do not add “View all comments” and “Hide comments” buttons under the group announcements if there are no comments

### ISSUE THREAD

**Issue by przemo20** [IsThereAnyDeal/AugmentedSteam#1979]
### Current Behavior (if any)
AS puts the two buttons mentioned in the title under every group announcement, no matter if there are any comments under them (and so you see a button saying, “View all 0 comments”). If you disable the extension, the “View all comments” button is not present – even Valve is not adding it in this scenario.

### Desired Behavior
If there are no comments, do not insert the “View all comments” and “Hide comments” buttons.

#### With Augmented Steam
![obraz](https://github.com/user-attachments/assets/57ca1b28-dcff-4f75-8fdc-4e16a45e9dea)

#### Vanilla
![obraz](https://github.com/user-attachments/assets/93f4b9d6-e77a-479a-bbaf-9c2bebb75751)

### Specifications

  - Augmented Steam Version: 3.1.1
  - Browser Version: Mozilla Firefox 128.0 (64-bit)

**Comment by candela97** [comment]
It's been like this for a while #488. Seems pretty harmless?

**Comment by przemo20** [comment]
Yeah, it is not that big of a deal. Just saying that it could be improved, since those buttons take additional space for no reason (in case of announcements without comments).
